### ISSUE TITLE
Uncaught TypeError: Cannot read property 'html' of undefined

### ISSUE THREAD

**Issue by germanocorrea** [leny/atom-w3c-validation#3]
[Enter steps to reproduce below:]

1.I tried to validate the CSS below:

``` css
body {
    background-color:white;
    font-family: 'Ubuntu Light', sans-serif;
}

header {

}
nav[aria-label="Skiplinks"] a{
    text-decoration:none;
    color:white;
}
nav[aria-label="Skiplinks"] li {
    color:white;
    background-color:black;
    display:none;
}
nav[aria-label="Skiplinks"] li:focus, li:active {
    color:white;
    background-color:black;
    display:inline;
}

nav ul li{
    list-style-type:none;
    display:inline;
}

.image_foto {
    width: 10em;
    margin:1em;
    border-radius:100%;
}
.trabImg {
    position:left;
    width:290px;
    border-radius:15px;
    box-shadow:0px 1px 3px black;
    display:block;
}
```
1. And them I got the error...

**Atom Version**: 0.168.0
**System**: linux 3.16.0-31-generic
**Thrown From**: [w3c-validation](https://github.com/leny/atom-w3c-validation) package, v0.1.3
### Stack Trace

Uncaught TypeError: Cannot read property 'html' of undefined

```
At events.js:85

TypeError: Cannot read property 'html' of undefined
  at /home/germanocorrea/.atom/packages/w3c-validation/node_modules/w3cvalidator/lib/w3cjs.js:105:29
  at Parser.<anonymous> (/home/germanocorrea/.atom/packages/w3c-validation/node_modules/w3cvalidator/node_modules/xml2js/lib/xml2js.js:429:18)
  at Parser.emit (events.js:107:17)
  at Object.onerror (/home/germanocorrea/.atom/packages/w3c-validation/node_modules/w3cvalidator/node_modules/xml2js/lib/xml2js.js:285:26)
  at emit (/home/germanocorrea/.atom/packages/w3c-validation/node_modules/w3cvalidator/node_modules/xml2js/node_modules/sax/lib/sax.js:615:33)
  at error (/home/germanocorrea/.atom/packages/w3c-validation/node_modules/w3cvalidator/node_modules/xml2js/node_modules/sax/lib/sax.js:644:3)
  at strictFail (/home/germanocorrea/.atom/packages/w3c-validation/node_modules/w3cvalidator/node_modules/xml2js/node_modules/sax/lib/sax.js:662:22)
  at closeTag (/home/germanocorrea/.atom/packages/w3c-validation/node_modules/w3cvalidator/node_modules/xml2js/node_modules/sax/lib/sax.js:845:7)
  at Object.write (/home/germanocorrea/.atom/packages/w3c-validation/node_modules/w3cvalidator/node_modules/xml2js/node_modules/sax/lib/sax.js:1294:29)
  at Parser.exports.Parser.Parser.parseString (/home/germanocorrea/.atom/packages/w3c-validation/node_modules/w3cvalidator/node_modules/xml2js/lib/xml2js.js:444:31)
  at Parser.parseString (/home/germanocorrea/.atom/packages/w3c-validation/node_modules/w3cvalidator/node_modules/xml2js/lib/xml2js.js:6:61)
  at Object.exports.parseString (/home/germanocorrea/.atom/packages/w3c-validation/node_modules/w3cvalidator/node_modules/xml2js/lib/xml2js.js:476:19)
  at parseCssValidation (/home/germanocorrea/.atom/packages/w3c-validation/node_modules/w3cvalidator/lib/w3cjs.js:103:23)
  at /home/germanocorrea/.atom/packages/w3c-validation/node_modules/w3cvalidator/lib/w3cjs.js:81:29
  at Request.callback (/home/germanocorrea/.atom/packages/w3c-validation/node_modules/w3cvalidator/node_modules/superagent/lib/node/index.js:630:3)
  at Request.<anonymous> (/home/germanocorrea/.atom/packages/w3c-validation/node_modules/w3cvalidator/node_modules/superagent/lib/node/index.js:131:10)
  at Request.emit (events.js:107:17)
  at IncomingMessage.<anonymous> (/home/germanocorrea/.atom/packages/w3c-validation/node_modules/w3cvalidator/node_modules/superagent/lib/node/index.js:773:12)
  at IncomingMessage.emit (events.js:129:20)
  at _stream_readable.js:907:16
  at process._tickCallback (node.js:386:11)

```
### Commands

```
     -3:02.1 core:cancel (atom-text-editor.editor)
     -2:58.8 find-and-replace:show (atom-text-editor.editor)
     -2:55.6 core:backspace (atom-text-editor.editor.mini)
     -2:14.0 editor:consolidate-selections (atom-text-editor.editor)
     -2:14.0 core:cancel (atom-text-editor.editor)
  2x -2:12.9 core:save (atom-text-editor.editor)
     -1:07.5 command-palette:toggle (atom-text-editor.editor.is-focused)
  3x -1:04.2 core:backspace (atom-text-editor.editor.mini)
     -1:00.8 core:select-all (atom-text-editor.editor.mini)
     -1:00.6 core:backspace (atom-text-editor.editor.mini)
     -0:59.8 editor:consolidate-selections (atom-text-editor.editor.mini)
     -0:59.8 core:cancel (atom-text-editor.editor.mini)
     -0:58.5 command-palette:toggle (atom-text-editor.editor.is-focused)
     -0:57.2 core:confirm (atom-text-editor.editor.mini)
     -0:57.2 settings-view:open (atom-text-editor.editor.is-focused)
     -0:00.0 core:save (atom-text-editor.editor.is-focused)
```
### Config

``` json
{
  "core": {
    "themes": [
      "seti-ui",
      "seti-syntax"
    ]
  }
}
```
### Installed Packages

``` coffee
# User
.git, vundefined
autoclose-html, v0.15.0
color-picker, v1.4.4
html-entities, v0.3.1
language-svg, v0.8.0
linter, v0.12.0
linter-clang, v2.25.1
linter-coffeelint, v0.1.7
linter-csslint, v0.0.11
linter-jshint, v0.1.0
linter-jsxhint, v0.1.1
linter-php, v0.0.11
linter-pylint, v0.2.1
linter-rubocop, v0.2.2
linter-tidy, v1.0.0
minimap, v3.5.6
seti-syntax, v0.3.3
seti-ui, v0.6.3
w3c-validation, v0.1.3

# Dev
No dev packages
```

/cc @atom/core

**Comment by leny** [comment]
Hi, thanks for the report.

I will try to investigate this tomorrow.

**Comment by leny** [comment]
Hi, sorry for the long delay. I can't reproduce the error since some corrections made to fix the API v1.0 deprecation warnings.

Seems to be fixed in `0.2.0`. Thanks for the report.
